File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var smallerNumbersThanCurrent = function(nums) {
    let array = [];
    
    nums.forEach(num => {
        let count = 0;
        
        for (let i = 0; i < nums.length; i++) {
            if (nums[i] < num) count = count + 1;
        }
        
        array.push(count);
    });
    
    return array;
};

File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.go
func smallerNumbersThanCurrent(nums []int) []int {
    
    var list []int
    
    for _, num := range nums {
        count := 0
        
        for i, _ := range nums {
            if nums[i] < num {
                count = count + 1
            }
        }
        
        list = append(list, count)
    }
    
    return list
}